templates/common/on-prem/units/on-prem-dnsmasq.yaml

```diff
@@ -0,0 +1,29 @@
+name: on-prem-dnsmasq.service
+enabled: true
+contents: |
+  [Unit]
+  Description=Run dnsmasq to provide local
+  Before=kubelet.service crio.service
+  [Service]
+  ExecStart=/bin/bash -c " \
```

Comment:
Perhaps using this as an `ExecStartPre` and then using `ExecStart=/usr/bin/dnsmasq -k`?

Reply:
Makes sense. This is a holdover from the nodeip-configuration service I stole the pattern from, but in that case the podman call is the only thing being run.